simulators/totalpower/__init__.py

```diff
@@ -2,20 +2,20 @@
 from math import modf
 from SocketServer import ThreadingTCPServer
 from simulators.common import ListeningSystem
-
+import socket, random
+import threading
 
 servers = [(('0.0.0.0', 11500), (), ThreadingTCPServer, {})]
 
-
 class System(ListeningSystem):
 
     tail = [b'\x0A', b'\x0D']  # NEWLINE and CR
 
     ack = b'ack\n'
     nak = b'nak\n'
 
-    channels = 14
-
+    channels = 4 # changed for totalpower (instead of 14)
```

Comment:
Maybe is better to set this parameter dynamically inside the constructor, I'll handle this before merging